Dockerfile

```diff
@@ -5,6 +5,9 @@ FROM docker.io/library/node:16.13.2 as build
 
 ARG BASE_URL_PLACEHOLDER
 
+# Skip puppeteer download since chromium isn't used by the project.
+ARG PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
```

Comment:
Thanks for that! For info: one of dev-dependency needs that, but we only use from that package only some infos from its `package.json`, so `puppeteer` is needed.